target/linux/ar71xx/image/generic.mk

```diff
@@ -32,6 +32,16 @@ define Build/mkwrggimg
 	mv $@.imghdr $@
 endef
 
+define Build/mkdapimg2
+	$(STAGING_DIR_HOST)/bin/mkdapimg2 \
+		-i $@ -o $@.new \
+		-s $(DAP_SIGNATURE) \
+		-v LEDE-$(firstword $(subst -, ,$(REVISION))) \
```

Comment:
Change "LEDE" to "OpenWrt" here.

Reply:
Ops :) Missed that!

Reply:
Even better would be if you use the $(VERSION_DIST) variable. Have a look at https://git.openwrt.org/43b432050c73a233a89012f13f01e81de1629611 for reference.

Reply:
Actually, use `$(VERSION_DIST)`

Reply:
Done :)